.vitepress/theme/styles/enhancements.css

```
@@ -0,0 +1,961 @@
+}
+
+/* Enhanced inline code */
+.VPDoc :not(pre) > code {
+  background: var(--vp-c-brand-soft);
+  color: var(--vp-c-brand-1);
+  padding: 0.25rem 0.5rem;
+  border-radius: 6px;
+  font-weight: 500;
+  font-size: 0.9em;
+  border: 1px solid var(--vp-c-brand-1);
+  border-opacity: 0.2;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)